webknossos/webknossos/utils.py

```diff
@@ -147,14 +148,22 @@ def setup_logging(args: argparse.Namespace) -> None:
     )
 
 
-def add_verbose_flag(parser: argparse.ArgumentParser) -> None:
+def add_silent_flag(parser: argparse.ArgumentParser) -> None:
     parser.add_argument(
         "--silent", help="Silent output", dest="verbose", action="store_false"
     )
 
     parser.set_defaults(verbose=True)
 
 
+def add_verbose_flag(parser: argparse.ArgumentParser) -> None:
+    parser.add_argument(
+        "--verbose", help="Verbose output", dest="verbose", action="store_true"
+    )
+
+    parser.set_defaults(verbose=False)
```

Comment:
So, depending on which utility function a module uses, either `--verbose` OR `--silent` is available? I imagine this to be tedious to use, since it's hard to remember. Why not always make both args available, but only differ on the default? So, one function to add both flags which accepts a "default_verbose" boolean parameter.

Reply:
I agree that it would be better to decide whether to have verbose on by default or not uniformly for all CLI tasks.
I'd vote for silent by default (which is now possible with the new progress bars).